code/datums/disease.dm

```diff
@@ -209,9 +209,12 @@ var/list/diseases = typesof(/datum/disease) - /datum/disease
 		return 1
 	return 0
 
+/datum/disease/proc/AffectSpeech(var/datum/speech/speech)
```

Comment:
This functionality already exists for several viruses such as the hangman's symptom etc.

Reply:
Viruses and diseases are separate entities, as far as I am aware. Diseases had no such feature.